src/main.rs

```diff
@@ -196,6 +196,20 @@ struct CLI {
     )]
     file_path: Option<PathBuf>,
 
+    #[structopt(
+        name = "negative-file",
+        value_name = "FILE",
+        short,
```

Comment:
The `short` setting should only be used for very obvious cases. I would remove it for this option. Again, I can do this myself.

Reply:
done